Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:2
+
+RUN apt-get -y update \
+ && apt-get -y install libgeos-dev \
+                       libpq-dev \
+                       python-pil \
+                       libmapnik2.2 \
```

Comment:
Looks like:

- https://github.com/tilezen/vector-datasource/wiki/Mapzen-Vector-Tile-Service

Has reference to Mapnik in error, compared to requirement file:

- https://github.com/tilezen/tileserver/blob/master/requirements.txt

So update the Wiki?